src/scenic/core/regions.py

```diff
@@ -1417,6 +1418,7 @@ def intersect(self, other, triedReversed=False):
             # Other region is a mesh volume. We can extract the mesh to perform boolean operations on it
             other_mesh = other.mesh
 
+            # If dimensions are prov
```

Comment:
Incomplete comment

Reply:
Not sure what this was but it doesn't seem necessary. Removing it.